rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -195,9 +195,12 @@ s32 cellHddGameExitBroken()
 }
 
 
-s32 cellGameDataGetSizeKB()
+s32 cellGameDataGetSizeKB(vm::ptr<u32> size)
 {
-	fmt::throw_exception("Unimplemented" HERE);
+	if (size) {
+		*size = 0;
```

Comment:
Totally agree. Didn't find any info on this function in docs, still kind of mystery for me what it should be doing.